USAGE.md

```diff
@@ -405,7 +405,7 @@ role_arn = arn:aws:iam::33333333333:role/role2
 mfa_serial = arn:aws:iam::111111111111:mfa/tom
 ```
 
-Be sure to specify the `mfa_serial` for the source profile (in the above example `tom`) so that aws-vault can match the common `mfa_serial`.
+For aws-vault <=v4, be sure to specify the `mfa_serial` for the source profile (in the above example `tom`) so that aws-vault can match the common `mfa_serial`.
```

Comment:
Hmm I don't think the version of aws-vault has any bearing on this. Specifying the mfa_serial in multiple places is required for aws-vault to determine if chaining is possible

Reply:
Sure thing, I got that from extrapolating from https://github.com/99designs/aws-vault/pull/1101/files/a03c62bdcff2a324adb364001900fd4a9f06ae8b#diff-58a825c70bccfbe691d7ddbddb133275f4f657de6aeb4a502fa9fce279f8bf8dR414, but I can remove the changes to this line if useful.

Reply:
Ok, reverted.